REQUIRE

```diff
@@ -1 +1,2 @@
 julia 0.4
+Compat 0.7.14+
```

Comment:
Can you avoid doing the `+` on these, just use the next tag version even though it doesn't exist yet? This may be a correct usage of the `+` but the more widespread it gets the more people will get the wrong impression and use it incorrectly.